### PR TITLE
selfdrive: Add Highlander 2018 LE

### PR DESCRIPTION
**Pull request by dougsland**
Add new fingerprint for Highlander

### CODE REVIEW

**Comment by rbiasini** [comment]
Thanks. could you please share a cabana route where you were running stock system so we can verify the fingerprint? You can DM it to me on Discord.

**Review by arne182 — APPROVED** [review]
No conflicts and these signals added 1984: 8, 1988: 8, 1992: 8, 1996: 8

File: selfdrive/car/toyota/values.py [hunk 1]
```diff
@@ -136,7 +136,7 @@ def check_ecu_msgs(fingerprint, ecu):
...
+    36: 8, 37: 8, 114: 5, 119: 6, 120: 4, 170: 8, 180: 8, 186: 4, 238: 4, 426: 6, 452: 8, 464: 8, 466: 8, 467: 8, 544: 4, 545: 5, 550: 8, 552: 4, 608: 8, 610: 5, 705: 8, 725: 2, 740: 5, 800: 8, 849: 4, 871: 2, 896: 8, 916: 3, 921: 8, 944: 8, 945: 8, 951: 8, 955: 8, 956: 8, 979: 2, 1008: 2, 1017: 8, 1020: 8, 1042: 8, 1044: 8, 1056: 8, 1059: 1, 1114: 8, 1161: 8, 1162: 8, 1163: 8, 1176: 8, 1177: 8, 1178: 8, 1179: 8, 1180: 8, 1181: 8, 1182: 8, 1183: 8, 1189: 8, 1190: 8, 1191: 8, 1192: 8, 1196: 8, 1197: 8, 1198: 8, 1199: 8, 1206: 8, 1212: 8, 1235: 8, 1279: 8, 1408: 8, 1409: 8, 1410: 8, 1552: 8, 1553: 8, 1554: 8, 1556: 8, 1557: 8, 1561: 8, 1562: 8, 1568: 8, 1569: 8, 1570: 8, 1571: 8, 1572: 8, 1592: 8, 1593: 8, 1595: 8, 1599: 8, 1745: 8, 1779: 8, 1872: 8, 1880: 8, 1984: 8, 1988: 8, 1992: 8, 1996: 8
```
Comment (dougsland) [inline selfdrive/car/toyota/values.py]: The command message is bigger from previous commit but I have noticed Lane Departure Alert Malfunction and Headlight System Malfunction alert come very often. The previous command work out of box. I would cherry pick the previous commit but here both messages I got.

**Comment by dougsland** [comment]
> Add new fingerprint for Highlander

Just cloned a fresh repo and tried again this fingerprint it works. I would merge it for now and make some Highlander users happy. Thanks!

**Comment by ErichMoraga** [comment]
Thanks @dougsland for the submission.  Please DM @rbiasini a cabana share link for a route where you were running stock ADAS (Giraffe DIP 1001) so Comma can verify the fingerprint?

**Comment by dougsland** [comment]
> Thanks @dougsland for the submission. Please DM @rbiasini (on Discord) a cabana share link for a route where you were running stock ADAS (Giraffe DIP 1001) so Comma can verify the fingerprint. Also, let us know if your car has any modifications like an aftermarket security system, remote start, or infotainment.

Hey @ErichMoraga. Sent him, I can send again or to you. Fell free to reach me at discord too.

**Comment by rbiasini** [comment]
This is merged internally with a longer fingerprint. If you waited long enough with, you could actually find 98 messages.

**Pull request closed without merging**